app/routes.py

```diff
@@ -1,2 +1,212 @@
-from flask import Blueprint
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime
+import requests
 
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+def customer_response_helper(customer):
+    return {
+            "id": customer.customer_id,
+            "name": customer.name,
+            "registered_at": customer.register_at,
+            "postal_code": customer.postal_code,
+            "phone": customer.phone_number,
+            "videos_checked_out_count": 0
+        }
+
+def video_response_helper(video):
+    return {
+            "id": video.video_id,
+            "title": video.title,
+            "release_date": video.release_date,
+            "total_inventory": video.total_inventory,
+            "available_inventory": video.available_inventory
+        }
+
+def rentals_response_helper(new_rental, customer, video):
+    return {
+            "customer_id": new_rental.customer_id,
+            "video_id": new_rental.video_id,
+            "due_date": new_rental.due_date,
+            "videos_checked_out_count": customer.videos_checked_out_count,
+            "available_inventory": video.available_inventory
+        }
+
+@customers_bp.route("", methods = ["GET"])
+def get_customers():
+    customers = Customer.query.all()
+    customers_response = []
+    if customers is None:
+        return make_response([])
+    for customer in customers:
+        customers_response.append(customer_response_helper(customer))
+    return jsonify(customers_response)
+
+@customers_bp.route("/<customer_id>", methods = ["GET"])
+def get_one_customer(customer_id):
+    customer = Customer.query.get_or_404(customer_id)
+    return customer_response_helper(customer)
+
+@customers_bp.route("", methods = ["POST"])
+def post_customers():
+    request_body = request.get_json()
+    if "name" not in request_body or "postal_code" not in request_body or "phone" not in request_body:
+        return make_response({"details": "Invalid data"}, 400)
+    else:
+        new_customer = Customer(name=request_body["name"], postal_code=request_body["postal_code"], phone_number=request_body["phone"])
+        db.session.add(new_customer)
+        db.session.commit()
+        return make_response({
+            "id": new_customer.customer_id,
+            "name": new_customer.name,
+            "postal_code": new_customer.postal_code,
+            "phone": new_customer.phone_number
+        }, 201)
+
+@customers_bp.route("/<customer_id>", methods = ["PUT"])
+def update_customer(customer_id):
+    customer = Customer.query.get(customer_id)
+    if customer is None:
+        return make_response({"details": "Customer not found"}, 404)
```

Comment:
here we could also use `get_or_404()`

```suggestion
    customer = Customer.query.get_or_404(customer_id)
```